src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/data/ReportRepositoryTest.java

```diff
@@ -20,9 +20,9 @@
 
 @ActiveProfiles({"integration-test", "db-test"})
 @SpringBootTest
-public class EnvelopeSummaryRepositoryTest {
+public class ReportRepositoryTest {
```

Comment:
perhaps just keep as it was or `EnvelopeSummaryReportTest` - class is big and can be bigger. Let's keep each report tested separately